apps/management/next.config.js

```diff
@@ -6,6 +6,19 @@ const withImages = require('next-images')
 
 const nextConfig = {
   future: { webpack5: true },
+  webpack: (config) => {
```

Comment:
Do we need these? AFAIK Next already has a setup for this

Reply:
did get errors, but probs not needed if remove above (as mentioned) and use defaults, ill try it